tests/core/views/test_api.py

```diff
@@ -170,16 +170,15 @@ async def test_pay_invoice_invoicekey(client, invoice, inkey_headers_from):
     assert response.status_code >= 300  # should fail
 
 
-# check POST /api/v1/payments: payment with admin key [should pass]
+# check POST /api/v1/payments: payment with admin key, trying to pay twice [should fail]
 @pytest.mark.asyncio
-@pytest.mark.skipif(is_regtest, reason="this only works in fakewallet")
 async def test_pay_invoice_adminkey(client, invoice, adminkey_headers_from):
     data = {"out": True, "bolt11": invoice["payment_request"]}
     # try payment with admin key
     response = await client.post(
         "/api/v1/payments", json=data, headers=adminkey_headers_from
     )
-    assert response.status_code < 300  # should pass
+    assert response.status_code > 300  # should fail
```

Comment:
Is this a related change?

Reply:
yes the test fails, because it tries to pay an internal invoice that is not pending.

Reply:
this was also the reason i skipped this testcase for regtest, because before the paid check, it failed there aswell.